x/cert/keeper/keeper_test.go

```diff
@@ -196,11 +191,11 @@ func (suite *KeeperTestSuite) TestCertificate_Delete() {
 				},
 			},
 			errArgs{
-				shouldPass: false,
+				shouldPass: true,
 				contains:   "",
 			},
 		},
-		{"Certificate(1) Delete: Add Three Delete the Second One",
+		{"Certificate(2) Delete: Add Three Delete the Second One",
```

Comment:
The numbers are intended to represent the number of certificates, but I guess you don't need to conform to it.